src/derivation.js

```diff
@@ -25,6 +25,10 @@ const modes = Object.freeze({
   // this is "default" by convention
   "": {},
 
+  stellar: {
+    overridesDerivation: "44'/148'/<account>'",
+    tag: "stellar"
```

Comment:
I don't think the tag is correct. Do you have a hint where this derivation scheme would be used? If not, maybe we call this legacy.

stellar on bip44 is standard / used by any app today?

Reply:
Right, remember when I wrote most of this code I had no idea what anything was. Tag can be removed then since we're not using bip44 and it's not really legacy